README.md

```diff
@@ -237,6 +237,40 @@ instances:
 
 **Note**: See the sample [mcache.d/conf.yaml][22] for all available configuration options.
 
+#### Using the prerun.sh script to dynamically change the integration configuration
+
+If you have configuration details stored in environment variables (like database configuration or secrets), you can use the [prerun.sh script](#prerun-script) to dynamically add those to your Datadog Agent configuration before the Agent starts.
+
+For example, to enable the Postgres integration, the file `datadog/conf.d/postgres.d/conf.yaml` could be added with placeholders at the root of your application (or `/$DD_HEROKU_CONF_FOLDER/conf.d/mcache.d/conf.yaml` if you have changed this [configuration option](#configuration)):
```

Comment:
I know the docs team prefers to have directions be direct (i.e. no "could"). So more like:
` For example, to enable the Postgres integration, copy the datadog/conf.d/postgres.d/conf.yaml to the root of your application and set the values to placeholders` ... `Then use the prerun.sh script to replace the place holders with actual values...`